config.go

```diff
@@ -19,6 +19,10 @@ var config struct {
 		MessageBird struct {
 			AccessKey string `yaml:"access_key"`
 		}
+		Exotel struct {
+			AccountSID string `yaml:"sid"`
```

Comment:
Can you change this to `yaml:"account_sid"`, or even to `SID string`, so it's consistent?